src/functionalTest/resources/features/common/data-store/FT_NoCCaseType/snippets/Questions_Response.td.json

```diff
@@ -0,0 +1,40 @@
+{
+  "_guid_": "Challenge_Questions_Response",
```

Comment:
Should be moved into MCA common folder (i.e.  `src/functionalTest/resources/features/common/mca`) as it is a snippet of a call to an MCA API not a data-store one.